src/main/java/com/minecolonies/coremod/entity/pathfinding/PathJobMoveAwayFromLocation.java

```diff
@@ -55,7 +68,29 @@ public PathJobMoveAwayFromLocation(final World world, @NotNull final BlockPos st
         dx *= scalar;
         dz *= scalar;
 
-        heuristicPoint = new BlockPos(start.getX() + (int) dx, start.getY(), start.getZ() + (int) dz);
+        int randomValue = rand.nextInt(4);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)